packages/odata-common/src/filter/filterable.ts

```diff
@@ -47,21 +61,10 @@ export type Filterable<
  *  .filter(filterExp1, filterExp2);
  * ```
  * @typeparam EntityT - Type of the entity filter on.
- * @param expressions - Filterables to be combined with logical `and`.
+ * @param first - The first [[Filterable]]s to be combined with.
+ * @param rest - The rest of the [[Filterable]]s to be combined with.
```

Comment:
[q] Have you checked what happens if you add docs to the other overloads? I think the first two are the actual API and this one is basically just the combined helper to allow both variadic arguments and arrays.
[nth] Intuitively, I think the wording is incorrect, but I am not sure:
```suggestion
 * @param first - The first [[Filterable]]s combine.
 * @param rest - The rest of the [[Filterable]]s to combine.
```

Reply:
To make the doc visible, the doc should be put on the overloads, instead of the combined one.
However, lint only checks the combined one..

Reply:
That's unforunate, but now I can see that this is only documented for one overload... Shouldn't we do this for all overloads (or at least the 2 ones that are not just helper functions?)